service/app/controllers/usermanagement/validator/UserSelfMigrationReqValidator.java

```diff
@@ -0,0 +1,37 @@
+package controllers.usermanagement.validator;
+
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.request.BaseRequestValidator;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class UserSelfMigrationReqValidator extends BaseRequestValidator {
```

Comment:
UserSelfMigrationReqValidator to ShadowUserMigrateReqValidator

Reply:
@manzarul done